app/contributing/style-guide.md

```diff
@@ -146,11 +177,7 @@ When deciding which icon to use for a doc, use the following guidelines:
 - Don't use link titles like "Read more" and "Click here". Instead, write descriptive titles that properly detail what content is accessible by clicking the link.
 - If the linked content is a larger area like a panel, add a `title` attribute that describes the linked content to the `a` tag.
 
-## Reference style guides
+## Reference style guide
 
-- [Valero Style Guide](https://velero.io/docs/v1.5/style-guide/#inline-code-formatting)
-- [Splunk Style Guide](https://docs.splunk.com/Documentation/StyleGuide/current/StyleGuide/Howtouse)
-- [Microsoft Style Guide](https://docs.microsoft.com/en-us/style-guide/welcome/)
-- [Barrie Byron’s admirably curated list of style guides](https://docs.google.com/document/d/1wAVt65UpgBJ4e_tzPCVnPHwOqYYtENuRkojDSq-7nK0/edit)
-- [Google Developers Guide](https://developers.google.com/style)
-- [Kubernetes](https://kubernetes.io/docs/contribute/style/style-guide/)
+Follow Kong's style guide whenever possible. However, you can also refer to other external style guides for style and formatting that our guide doesn't cover.
```

Comment:
```suggestion
Follow Kong's style guide whenever possible. However, we recommend using the [Google developer style guide](https://developers.google.com/style/) for style and formatting cases that our guide doesn't cover.
```

Reply:
I think it is more confusing to say 'other external style guides' than it is to recommend one explicitly.